ccf_transaction_processor/transaction_processor/enclave_registry.h

```diff
@@ -21,6 +21,52 @@ using namespace std;
 namespace ccf
 {
 
+  struct ContractEnclaveAttestationVerificationPolicy {
+    bool check_attestation;
+    string mrenclave;
+    string basename;
+    string ias_public_key;
+  };
+
+  DECLARE_JSON_TYPE(ContractEnclaveAttestationVerificationPolicy);
+  DECLARE_JSON_REQUIRED_FIELDS(ContractEnclaveAttestationVerificationPolicy,
+    check_attestation,
+    mrenclave,
+    basename,
+    ias_public_key);
+
+
+  struct ProofData{
+    vector<string> certificates;
+    string verification_report;
+    string signature;
+  };
+  DECLARE_JSON_TYPE(ProofData);
+  DECLARE_JSON_REQUIRED_FIELDS(ProofData,
+  certificates,
+  verification_report,
+  signature);
+
+  struct VerificationReport{
+    string epidPseudonym;
+    string id;
+    string isvEnclaveQuoteStatus;
+    string isvEnclaveQuoteBody;
+    int version;
+    string nonce;
+    string timestamp;
+  };
+  DECLARE_JSON_TYPE(VerificationReport);
+  DECLARE_JSON_REQUIRED_FIELDS(VerificationReport,
+    epidPseudonym,
```

Comment:
Same style request. Line up all of them.

Reply:
thank you, done